itest_src/datasets/case02/queries/1004.2_varObjects.sql

```diff
@@ -3,7 +3,6 @@
 
 SELECT objectId
 FROM   Object
-WHERE  scisql_s2PtInCircle(ra_PS, decl_PS, 1.5, 3, 1) = 1 -- noQserv
--- withQserv WHERE qserv_areaspec_circle(1.5, 3, 1)
+WHERE  scisql_s2PtInCircle(ra_PS, decl_PS, 1.5, 3, 1)
```

Comment:
I think it would probably be good to have test cases for both with _and_ without `=1` versions of this now non `qserv_areaspec_circle` query.

Also, you do want the `AND variability ...` clause here; now that the conditionals are gone, should you remove the `--`?

Reply:
We do need a test with a more complex expression as this was discussed earlier in the team's Slack channel.